app/controllers/api/v1/salons_social_networks_controller.rb

```diff
@@ -0,0 +1,59 @@
+module Api
+  module V1
+    class SalonsSocialNetworksController < ApplicationController
+      before_action :set_salons_social_network, only: %i[update destroy]
+
+      # GET /salons/1/salons_social_networks
+      def index
+        salons_social_networks = SalonsSocialNetwork.where(salon_id: params[:salon_id]).all
+        render json: salons_social_networks
+      end
+
+      # GET /salons/1/salons_social_networks/1
+      def show
+        render json: set_salons_social_network
```

Comment:
You can use just `@salons_social_network` object because you are setting up in before_action helper